### PR TITLE
Migrate readme yaml

### PR DESCRIPTION
**Pull request by solairerove**
## what
- regenerate readme md
- add description key in outputs values
- add makefile
- ignore harness
- prepare to open discussions on review

## why
- need to add categories and tags so we can pull them into the documentation

### CODE REVIEW

**Comment by solairerove** [comment]
write plz necessary changes to add

File: README.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+#   module "user_data_nfs" {
+#     source = "git::https://github.com/cloudposse/terraform-template-user-data-assests.git?ref=master"
+#     namespace                  = "${var.namespace}"
+#     name                       = "${var.name}"
+#     stage                      = "${var.stage}"
+#     dir                        = "${var.dir}"
+#     host                       = "${var.host}"
+#   }
+#   ```
+
```
Comment (osterman) [inline README.yaml:72]: Let's add related projects. List all the `user-data` modules as "related". See other readme yaml for example.

**Review by osterman — CHANGES_REQUESTED** [review]
Almost perfect. Add related modules. See comment.

File: README.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+# related:
+  # - name: "terraform-aws-cloudfront-s3-cdn"
+    # description: "Terraform module to easily provision CloudFront CDN backed by an S3 origin"
+    # url: "https://travis-ci.org/cloudposse/terraform-aws-cloudfront-s3-cdn"
+
+# Short description of this project
+description: |-
+  Short description of this project
+
+# How to use this project
```
Comment (osterman) [inline README.yaml]: Let's uncomment this and make a working example.

**Comment by osterman** [comment]
Fix CI/CD

**Comment by osterman** [comment]
image


Run `terraform fmt` to fix

**Comment by solairerove** [comment]
i ran `terraform fmt` but it is still failing

File: docs/terraform.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+| name | Name  (e.g. `app` or `cluster`) | string | - | yes |
+| namespace | Namespace (e.g. `cp` or `cloudposse`) | string | - | yes |
+| ssh_user | User that used to execute backup cron | string | - | yes |
+| stage | Stage (e.g. `prod`, `dev`, `staging`) | string | - | yes |
+
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| policy_arn | description here |
```
Comment (osterman) [inline docs/terraform.md]: "description here" does not count as a description =P

File: outputs.tf [hunk 4]
```diff
@@ -1,7 +1,9 @@
 output "user_data" {
-  value = "${data.template_file.default.rendered}"
+  value       = "${data.template_file.default.rendered}"
+  description = "description here"
```
Comment (osterman) [inline outputs.tf]: Saerch other modules for a better example of `user_data` description

File: outputs.tf [hunk 5]
```diff
@@ -1,7 +1,9 @@
...
 }
 
 output "policy_arn" {
-  value = "${aws_iam_policy.default.arn}"
+  value       = "${aws_iam_policy.default.arn}"
+  description = "description here"
```
Comment (osterman) [inline outputs.tf]: Search other modules for a better example of `policy_arn` description

**Comment by solairerove** [comment]
@osterman resolve review issues

File: README.md [hunk 6]
```diff
@@ -1 +1,263 @@
...
+| name | Name  (e.g. `app` or `cluster`) | string | - | yes |
+| namespace | Namespace (e.g. `cp` or `cloudposse`) | string | - | yes |
+| ssh_user | User that used to execute backup cron | string | - | yes |
+| stage | Stage (e.g. `prod`, `dev`, `staging`) | string | - | yes |
+
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| policy_arn | IAM policy ARNы |
```
Comment (osterman) [inline README.md]:
```suggestion
| policy_arn | IAM policy ARNs |
```

File: docs/terraform.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+| name | Name  (e.g. `app` or `cluster`) | string | - | yes |
+| namespace | Namespace (e.g. `cp` or `cloudposse`) | string | - | yes |
+| ssh_user | User that used to execute backup cron | string | - | yes |
+| stage | Stage (e.g. `prod`, `dev`, `staging`) | string | - | yes |
+
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| policy_arn | IAM policy ARNы |
```
Comment (osterman) [inline docs/terraform.md]:
```suggestion
| policy_arn | IAM policy ARNs |
```

**Review by osterman — CHANGES_REQUESTED** [review]
See comments

**Comment by solairerove** [comment]
@osterman 
fixed

**Pull request closed without merging**